packages/store/src/StoreErrors.sol

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.24;
+
+import { ResourceId } from "./ResourceId.sol";
+
+// Errors include a stringified version of the tableId for easier debugging if cleartext tableIds are used
```

Comment:
I know this wasn't part of your PR, but thought we could improve/clarify this one:

```suggestion
// Errors include a human readable resource/table ID for better logging and debugging
```